person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    true if @age > 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age?
+  end
```

Comment:
- [ ] Please implement the `or` condition in this method, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=or%20if%20they%20have%20permission%20from%20parents.) requirement.
- [HINT] You can use the "or" operator `||` and pass in the `@parent_permission`.

Reply:
Thank you for your notes. 👍 💯